lib/google/gax/operation.rb

```diff
@@ -214,7 +214,7 @@ def cancel options: nil
         # Converts hash and nil to an options object
         options = ApiCall::Options.new options.to_h if options.respond_to? :to_h
 
-        @client.cancel_operation @grpc_op.name, options: options
+        @client.cancel_operation name: @grpc_op.name, options: options
```

Comment:
These were changed on the generated code side, I take it? It seems `name` would be a required parameter; why did we change this?

Reply:
This is part of the changes in the new generator. Client methods are not backwards compatible, as all field arguments are

Reply:
Oh, that's right, I forgot, these are client methods. Okay then, carry on. 👍